src/containers/Guide/GuideBlocks/index.tsx

```diff
@@ -1,108 +1,118 @@
 import React, {FC} from 'react';
+import {NavLink} from 'react-router-dom';
 
-import {A, DocContainer, DocImage, DocList, TableBorderGrid} from 'components';
+import {
+  A,
+  CalloutType,
+  DocCallout,
+  DocContainer,
+  DocImage,
+  DocList,
+  DocSubHeader,
+  TableBorderGrid,
+  TableVertical,
+} from 'components';
 
 import BalanceLockAndKey from './BalanceLockAndKey.png';
 import BlockDetails from './BlockDetails.png';
 import './Blocks.scss';
 
 const GuideBlocks: FC = () => {
   return (
-    <DocContainer className="GuideBlocks" title="Blocks" lastUpdated="07 Dec 2020">
+    <DocContainer className="GuideBlocks" title="Blocks" lastUpdated="17 Feb 2021">
       <p>
-        A block is a group of one or more transactions. There are often multiple transactions (usually fees) to
-        different recipients within a single block. We can inspect the structure of a block by first examining a
-        simplified overview below. In this example, Amy is sending 100 coins to her friend Brian.
+        A block is a group of one or more transactions. There are often multiple transactions (usually fees) with
+        different recipients within a single block. Let's inspect the structure of a block by first examining the
+        simplified scenario below.
       </p>
+      <DocSubHeader>Example</DocSubHeader>
+      <p>In this example, Amy sends 100 coins to Brian:</p>
 
       <TableBorderGrid
         rows={[
-          ['Coins being sent to Brian', '100'],
+          [<strong>Description</strong>, <strong>Coins</strong>],
+          ['Coins sent to Brian', '100'],
           ['Bank fees', <span className="GuideBlocks__text-light">2</span>],
           ['Validator fees', <span className="GuideBlocks__text-light">1</span>],
-          ['Total', '103'],
+          ['Total cost of transaction for Amy', '103'],
         ]}
       />
 
+      <p>The network must validate several aspects of this block and all related transactions, including:</p>
+      <TableVertical
+        altColors
+        rows={[
+          [<strong>Validation</strong>, <strong>Notes</strong>],
+          [
+            'Amy has signed the block',
+            'The term "signing" is preferred over "created" because Amy might sign (authorize) blocks that she did not originate, for example, as Amy adds items to her cart on a shopping website, this builds the block',
```

Comment:
i feel like this sentence can be improved? It is hard to understand while reading, maybe it's a run-on sentence (I failed English half the time, so ignore my comment if I am speaking nonsense)

Reply:
yeah, I agree. Maybe a native speaker would help us here. Unless we just keep the original one, which was ok. I will try to think why I made this change